src/ngx_postgres_util.c

```diff
@@ -61,10 +61,18 @@ ngx_postgres_upstream_finalize_request(ngx_http_request_t *r,
         u->resolved->ctx = NULL;
     }
 
+#if defined(nginx_version) && (nginx_version >= 1009001)
+    if (u->state && u->state->response_time) {
+#else
     if (u->state && u->state->response_sec) {
+#endif
         tp = ngx_timeofday();
+#if defined(nginx_version) && (nginx_version >= 1009001)
```

Comment:
I suggest merge these two `#if` into a single one. Also, we don't need the "tp" variable assignment code path for nginx 1.9.1+.